src/blts-fbdev-fute.c

```diff
@@ -66,7 +66,7 @@ void log_print_fb_fix_screeninfo(struct fb_fix_screeninfo *fsi)
 	       "mmio_len    = %u\n"		/* Length of Memory Mapped I/O  */
 	       "accel       = %u\n"		/* Indicate to driver which	*/
 	       /*  specific chip/card we have	*/
-	       "reserved[]  = [%u, %u, %u]\n"		/* Reserved for future compatibility */
+	       "reserved[]  = [%u, %u]\n"		/* Reserved for future compatibility */
```

Comment:
Maybe add the 'capabilities' that was added to the struct?
And baybe these reserved bits could be removed from the output? Though the size change serves as an indicator that something has been added.

Reply:
I'm not sure how much we need the whole component, especially when it hasn't been compiling properly for some time, so I'd limit to minimum changes. But yea, doesn't make much sense to print out reserved fields, so maybe I'll just remove that part.

Reply:
It's gone.